Update PyCGNS link

MicK7 opened the pull request:
PyCGNS moved to github.com

brtnfld commented:
@MicK7, could you review the current webpage built with this fork? I switched to using  sphinx_book_theme. I like the right-hand side menu, especially for navigating the APIs. I disabled the dark mode because I don't think it looks all that great, and I'm not a fan of the purple highlight with orange text. Other than that, I think it looks good.

I tried most of the other themes, and they either had an issue rendering the doxygen/breathe output or would not let me customize it without needing a lot of CSS overrides, and I'm not that versed in CSS.

I added the CGNS repo as a submodule, so you should use the following:

`git clone --recurse-submodules https://github.com/brtnfld/cgns-modern.github.io.git`

Then in `CGNS-ReST-site`, you can do `make html`

I still need to move over some particle content from the HTML pages.

MicK7 commented:
> other themes, and they either had an issue rendering the doxygen/breathe output or would not let me customize it without needing a lot of CSS overrides, and



> @MicK7, could you review the current webpage built with this fork? I switched to using sphinx_book_theme. I like the right-hand side menu, especially for navigating the APIs. I disabled the dark mode because I don't think it looks all that great, and I'm not a fan of the purple highlight with orange text. Other than that, I think it looks good.
> 
> I tried most of the other themes, and they either had an issue rendering the doxygen/breathe output or would not let me customize it without needing a lot of CSS overrides, and I'm not that versed in CSS.
> 
> I added the CGNS repo as a submodule, so you should use the following:
> 
> `git clone --recurse-submodules https://github.com/brtnfld/cgns-modern.github.io.git`
> 
> Then in `CGNS-ReST-site`, you can do `make html`
> 
> I still need to move over some particle content from the HTML pages.

Thanks. I will look into it as soon as possible.
Like you I would prefer a light color theme as the original website and not a dark one.

brtnfld commented:
Thanks. I'm also not sure the site map is needed on the landing page, but I'm not sure how to remove it without also removing the site map in the right-hand side panel.

MicK7 commented:
> Thanks. I'm also not sure the site map is needed on the landing page, but I'm not sure how to remove it without also removing the site map in the right-hand side panel.

I think it can be hidden by declaring the toctree hidden 
```rest
.. toctree::
  :maxdepth: 4
  :hidden:

  general
  ...
```

The new theme is pretty nice.

brtnfld commented:
Yes, that worked to hide the TOC. I'm also not sure if the Main Topics Table is needed, as it partially repeats the RHS menu and is somewhat disconnected from the topics in the RHS menu. I'm also still determining about the news section as it tends to get behind and needs to be updated.

MicK7 commented:
Maybe an item should be added to the TODO list:
- [x] Update Minutes

brtnfld commented:
@MicK7, I'm close to calling this "good enough, for now," with just a few tasks remaining. We should switch to this website as the main webpage, archive the old site (and ensure the search engines do not prioritize it over the current one), and archive the old webpage repository. This task would be nice to have before the release.

MicK7 commented:
Sounds good to me. Do we create a cgns-archive.github.io for the old site ?

brtnfld commented:
> Sounds good to me. Do we create a cgns-archive.github.io for the old site ?

We can rename the cgns.github.io to cgns-archive.github.io, then rename cgns-modern.github.io to cgns.github.io.

brtnfld commented on CGNS-ReST-site/source/standard/MLL/api/general_remarks.rst:
```diff
@@ -0,0 +1,229 @@
+is always a 64-bit integer, and ``cgsize_t`` will be either a 32-bit or 64-bit integer depending on how the library was built.
+Many of the C functions in the MLL have been changed to use ``cgsize_t`` instead of *int* in the arguments. These functions
+include any that may exceed the 2Gb limit of an int, e.g. zone dimensions, element data, boundary conditions, and connectivity.
+In Fortran, all integer data is taken to be *INTEGER\*4* for 32-bit and *INTEGER\*8* for 64-bit builds. 
```
@MicK7  I'm not entirely sure that the sentence is accurate or what specific data it refers to. INTEGERs are not promoted to 64-bit with enable 64-bit. I think it should be removed..

MicK7 replied:
I agree let's remove it. The first time I read it I though it was for cgsize_t type but the "all integer" is completly wrong.

brtnfld commented:
- [ ]  Add links back into the overview section

brtnfld commented:
@MicK7 , I think I am done with my updates. The PR is getting large, so any more updates should be made to the new PR. 
I think it is in a state that it can replace the current one as the default.

MicK7 commented:
> @MicK7 , I think I am done with my updates. The PR is getting large, so any more updates should be made to the new PR. 
> I think it is in a state that it can replace the current one as the default.

I agree. Let's do the replacement and update through new PRs after to polish the result.
